examples/slave/nuttx/xmc4800/README.md

```diff
@@ -1,12 +1,11 @@
 ### Quick start
 
 #### Environment setup
-- download and build NuttX project (see NuttX documentation for details)
-- copy the `defconfig` file into `nuttxspace/nuttx/boards/arm/xmc4/xmc4800-relax/configs/nsh/defconfig`
-- configure the NuttX project with `./tools/configure.sh -E -l xmc4800-relax:nsh` in NuttX folder.
-- install `arm-none-eabi-objcopy` // TODO more details
-- install `JLinkExe`              // TODO more details
-- use the `export_nuttx_archive.sh` to build and deploy your app. You will need to tweak the paths in the script to match your environment.
+- Install `JLinkExe`              // TODO more details
```

Comment:
JlinkExe can either come from https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack using the deb